packages/esm-patient-chart-app/src/constants.ts

```diff
@@ -6,3 +6,5 @@ export const moduleName = '@openmrs/esm-patient-chart-app';
 export const patientChartWorkspaceSlot = 'patient-chart-workspace-slot';
 export const patientChartWorkspaceHeaderSlot = 'patient-chart-workspace-header-slot';
 export const omrsDateFormat = 'YYYY-MM-DDTHH:mm:ss.SSSZZ';
+export const otzUuid = '159836AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA';
+export const otzOutPut = 'Opt out of OTZ';
```

Comment:
We don't need this hard coded values. These should be provided by the config file